[FF-588] - Update web_objects.md (#7960)
braze-inc/braze-docs: _docs/_api/objects_filters/messaging/web_objects.md

   "custom_uri": (optional, string) a web URL,
   "image_url": (optional, string) URL for image to show,
   "large_image_url": (optional, string) URL for large image, supported on Chrome Windows/Android,

>    "require_interaction": (optional, boolean) whether to require the user to dismiss the notification, supported on Firefox for Windows, Chrome, Edge, and Opera,

Author: Is it possible to link to the MDN documentation from within this code block?

Reviewer: ```suggestion
   "require_interaction": (optional, boolean) whether to require the user to dismiss the notification. for list of supported platforms, see: "https://developer.mozilla.org/en-US/docs/Web/API/Notification/requireInteraction#browser_compatibility",
```

Would something like this work? Alternatively, I can see if its against our style guide to use short URLs? So at least it won't be as long?

That being said, there's no way to do this `[MDN docs](url_string)` _within_ a codeblock on Braze Docs.

Reviewer: @wesleyorbin , just tagging you here in case the notification fell through the cracks. Let me know if my suggestion above works for you 👍🏽

Author: That suggestion works for me. Thank you!
